src/test/java/com/datadog/api/v2/client/api/SecurityMonitoringApiTest.java

```diff
@@ -171,7 +171,9 @@ public void rulesCrudTest() throws ApiException {
     @Test
     public void searchSignals() throws Exception {
         String uniqueName = getUniqueEntityName();
-        SecurityMonitoringRuleResponse rule = createRule(uniqueName);
+        createRule(uniqueName);
+        // Wait for the backend to start processing the rule
+        Thread.sleep(5000);
```

Comment:
Please skip those sleep calls when we use recorded cassettes.

Reply:
Fixed!